### ISSUE TITLE
Setting for file-conflict

### ISSUE THREAD

**Issue by Spielmops** [nextcloud/android#12178]
### Is your feature request related to a problem? Please describe.

I upload several files to my server, because they changed. And then there ist the bulk of notifications with "file-conflict, which file do you want to stay" ...

### Describe the solution you'd like

I wish there would be a setting, where I could set a default action for that.

Spielmops

**Comment by mattbell87** [comment]
+1 from me.

I have had a bunch of these notifications sitting on my phone for well over a month. It happened after switching my server to AIO. There are so many of them I just give up 😆. 

It would be great to see an "Apply to all" checkbox.

I may end up clearing out the photos/videos on my phone just so there's nothing to sync. Could be a workaround... not ideal though.

**Comment by MaciejIwan** [comment]
App could compare hash for conflicted files and skip if name && hashes are the same

**Comment by tovine** [comment]
I agree 100% with @MaciejIwan on this - if the file is identical then it should be automatically skipped.
This can for example be checked by first seeing if the size is the same, and then if the file size matches you could compare the checksum (so you don't have to spend time on that if the size is different).
